docs/img/omnisci/omnisci_import.svg

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why it has a "OmniSci Server" block? We don't run a standalone server, do we?.. I would suggest labelling it as "OmniSci engine" or "OmniSci library".

Reply:
> We don't run a standalone server, do we?

I think we actually do. In modin, we communicate with OmniSci through [`OmnisciServer` class](https://github.com/modin-project/modin/blob/cde6390a9e84df88cb1cf8bb675525737c4aaddf/modin/experimental/core/execution/native/implementations/omnisci_on_native/omnisci_worker.py#L38), which is a wrapper of `omniscidbe` entity called `PyDbEngine`, which is literally an OmniSci server.

However, let's ask a person, who knows for sure (@ienkovich). The picture we're discussing is this:
<details><summary>Pic</summary>

![image](https://user-images.githubusercontent.com/62142979/145213775-d87250a3-57b3-4dd4-bb11-7b4ed18edeff.png)

</details>

Reply:
@ienkovich, look at not only this question, but and at the whole changes too please.